src/global_scheduler/global_scheduler.h

```diff
@@ -39,10 +39,12 @@ typedef struct {
   UT_hash_handle hh;
 } scheduler_object_info;
 
+/* plasma_manager ip:port -> photon_db_client_id association entry. */
```

Comment:
sure, maybe we should add doxygen generation to the build, so it explicitly fails the build if the comments are not doxygen compliant. Since this is a header file (for which we expect to produce doxygen documentation), I agree with your comment placement suggestion.